app/scripts/session/sessionUtil.js

```diff
@@ -46,7 +46,13 @@
       var def = new $.Deferred();
       var _virtualCohort = $.extend(true, {}, virtualCohort_);
       _virtualCohort.filters = filters;
-      _virtualCohort.studies = cases;
+      
+      _virtualCohort.studies = cases.map(function(studyObj) {
+        var _study = $.extend(true, {}, studies_);
```

Comment:
If you manually assign id and samples to _study object. No need to do a deep copy anymore. `studies_` can even be removed

Reply:
yeah! true